catroid/src/org/catrobat/catroid/io/XStreamToSupportCatrobatLanguageVersion095AndBefore.java

```diff
@@ -412,10 +414,34 @@ private void modifyXMLToSupportUnknownFields(File file) {
 			modifyBrickLists(originalDocument);
 			modifyVariables(originalDocument);
 			checkReferences(originalDocument.getDocumentElement());
+
 			saveDocument(originalDocument, file);
 		}
 	}
 
+	private void updateLegoNXTFields(Document originalDocument) {
+
+		final String oldDriveMotors = "MOTOR_A_C";
+		final String newDriveMotors = "MOTOR_B_C";
+
+		final String oldMotorMoveBrickName = "legoNxtMotorActionBrick";
+		final String newMotorMoveBrickName = "legoNxtMotorMoveBrick";
+
+		NodeList motors = originalDocument.getElementsByTagName("motor");
+		for (int i = 0; i < motors.getLength(); i++) {
+			Node motor = motors.item(i);
+			if (motor.getTextContent().equals(oldDriveMotors)) {
+				motor.setTextContent(newDriveMotors);
+			}
+		}
+
+		NodeList motorMoveBricks = originalDocument.getElementsByTagName(oldMotorMoveBrickName);
+		for (int i = 0; i < motorMoveBricks.getLength(); i++) {
+			Node motorMoveBrick = motorMoveBricks.item(i);
+			originalDocument.renameNode(motorMoveBrick, motorMoveBrick.getNamespaceURI(), newMotorMoveBrickName);
+		}
+	}
+
```

Comment:
Nice! Those renames got handled correctly, but we still need to increment our Catrobat Language Version.
( + rename of the `XStreamToSupportCatrobatLanguageVersion095..` classes)